cluster/images/conformance/go-runner/tar_test.go

```diff
@@ -36,7 +36,7 @@ func TestTar(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	defer os.Remove(tmp)
+	defer os.RemoveAll(tmp)
```

Comment:
general comment, not specific to this test case: I'm curious if the error output here may help to uncover some issues with test synchronization. For instance, if some routine keeps writing to the file in this folder, having an error log from here may help to uncover this.

Reply:
Like here: https://github.com/kubernetes/kubernetes/blob/cb719f75aac98ca24224c563d80d1035079885a5/staging/src/k8s.io/client-go/util/certificate/certificate_store_test.go#L259-L262

Reply:
I also noticed that. AFAIK there are so many places(374/458) don't do the cleanup error check in unit test currently.

i might need another cleanup to do this i guess.

Reply:
Why wouldn't you add it in places you introduces the call to `Remove`? This PR requires so many approvals so doing this clean up twice will be challenging =)

It's not blocking from my perspective though